### ISSUE TITLE
[DevOps] Change instances of "https://env.vet360.va.gov/" to "https://env.vaprofile.va.gov/" in INT, QA, PPD, PRD environments

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#14188]
## Background

The following request came in from VA Profile on 9.21.20:

> VA Profile (known as Vet360 prior to 12/31/2019) has been completing compliance activities to remove any usage of the previous project name. Based on our reporting, we have observed that your team is still using the URL having our old name to call our endpoints for your program/project needs.  Here is the host name in the URL that you are using to call the endpoints for our services where the change needs to be made (change the instance of “vet360” to “vaprofile”):
>  
> - **Old (currently in use)**: https://env.vet360.va.gov/ …..
> - **New (needs to be used)**: https://env.vaprofile.va.gov / ….
> - **Environments (env)**: INT, QA, PPD, PRD
>  
> Partner Action(s):
> 1. Migrate to VA Profile URL -  VA Profile will remove instances of vet360 by the end of this calendar year (Dec 31, 2020)
> - Use our lower environments for testing first and report issues if any to VA Profile OM Group:  VAProfileOM@va.gov
> - Make URL change in PROD if no issues are encountered in the lowers
> 2. Inform us of the timeline when you are planning on changing the instance(s) in the lower and prod environments by Sept 30, 2020

TL;DR — VA Profile needs to make sure that all mentions of "Vet360" are removed from our code base as they are no longer going by "Vet360". This includes in the URLs listed above. We need to update all instances of "https://env.vet360.va.gov/" to "https://env.vaprofile.va.gov/" in INT, QA, PPD, PRD environments by no later than 12.31.20.

## Tasks

- [ ] Change all instances of "https://env.vet360.va.gov/" to "https://env.vaprofile.va.gov/" in lower environments
- [ ] Test in lower environments and reach out to VA Profile if there are issues
- [ ] If no issues, make updates in prod environment
- [ ] Inform Samara/VA Profile when this is complete

**Comment by Samara-Strauss** [comment]
@tskinn thank you for helping us find a DevOps person who can help us with this work. Let us know who might be helping. On our side of things, they can coordinate with @lihanli as needed.
